app/modules/configuration/resources.py

```diff
@@ -96,6 +97,11 @@ def get(self, target, path):
             data['response']['configuration'][
                 'site.adminUserInitialized'
             ] = User.admin_user_initialized()
+            site_settings = SiteSetting.query.filter_by(public=True).order_by('key')
+            ss_json = {}
+            for ss in site_settings:
+                ss_json[ss.key] = f'/api/v1/fileuploads/src/{str(ss.file_upload.guid)}'
```

Comment:
Site settings will include things other than files (PR275) so you need to be sure that this file_upload is not None first here